File: src/NetFxCSharp/FacessoConfig/frmMain.vb
Imports System.Collections.ObjectModel
Imports Facesso
Imports System.Data.SqlClient

Public Class frmMain

    Sub New()

        ' This call is required by the designer.
        InitializeComponent()

        ' Add any initialization after the InitializeComponent() call.
        Me.DoubleBuffered = True
    End Sub

    Protected Overrides Sub OnPaint(ByVal e As System.Windows.Forms.PaintEventArgs)
        MyBase.OnPaint(e)

        Dim g = e.Graphics

    End Sub

    Private Sub btnSetupDatabase_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnSetupDatabase.Click
        Dim locDbSetupWizard As New frmDbSetupWizard
        locDbSetupWizard.ShowDialog()
    End Sub

    Private Sub btnActivateFacesso_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnActivateFacesso.Click
        Dim locSetupWizard As New frmSetupWizard
        locSetupWizard.ShowDialog()
    End Sub

    Private Sub btnSetDatabaseInstance_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnSetDatabaseInstance.Click
        Dim locFrm As New ActiveDev.Data.SqlClient.ADDatabaseConnectionDialog()
        Dim locSqlConnBuilder As SqlConnectionStringBuilder = locFrm.GetConnectionBuilder()
        If locSqlConnBuilder IsNot Nothing Then
            RegistryHelper.SetConnectionString(locSqlConnBuilder.ToString)
        End If
    End Sub

    Private Sub btnUpdateSchema_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnUpdateSchema.Click

    End Sub
End Class

File: src/NetFxCSharp/FacessoConfig/frmMain.cs
using Facesso;
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Data;
using System.Data.SqlClient;
using System.Diagnostics;
using System.Drawing;
using System.Linq;
using System.Threading.Tasks;
using System.Windows.Forms;
using System.Xml.Linq;

namespace FacessoConfig
{
    public partial class frmMain
    {
        public frmMain()
        {
            // This call is required by the designer.
            InitializeComponent();
            // Add any initialization after the InitializeComponent() call.
            this.DoubleBuffered = true;
        }

        protected override void OnPaint(System.Windows.Forms.PaintEventArgs e)
        {
            base.OnPaint(e);
            var g = e.Graphics;
        }

        private void btnSetupDatabase_Click(System.Object sender, System.EventArgs e)
        {
            frmDbSetupWizard locDbSetupWizard = new frmDbSetupWizard();
            locDbSetupWizard.ShowDialog();
        }

        private void btnActivateFacesso_Click(System.Object sender, System.EventArgs e)
        {
            frmSetupWizard locSetupWizard = new frmSetupWizard();
            locSetupWizard.ShowDialog();
        }

        private void btnSetDatabaseInstance_Click(System.Object sender, System.EventArgs e)
        {
            ActiveDev.Data.SqlClient.ADDatabaseConnectionDialog locFrm = new ActiveDev.Data.SqlClient.ADDatabaseConnectionDialog();
            SqlConnectionStringBuilder locSqlConnBuilder = locFrm.GetConnectionBuilder();
            if (locSqlConnBuilder != null)
            {
                RegistryHelper.SetConnectionString(locSqlConnBuilder.ToString());
            }
        }

        private void btnUpdateSchema_Click(System.Object sender, System.EventArgs e)
        {
        }
    }
}